docs/content/content/content-styles.mdx

```diff
@@ -5,6 +5,126 @@ complete: no
 order: 4
 ---
 
+## Abbreviations
+
+An abbreviation is a shortened form of a word. It contains the first letter or more of a word generally. They don’t include a word’s last letter. 
+
+Limit the use of abbreviations – or shortened words – in the Export Service. Users may not know their meaning or find them hard to understand.  
+ 
+Only use an abbreviation in limited circumstances. That is, if: 
+
+it’s a commonly used word that research shows users are familiar with. If there’s any doubt, don’t use an abbreviation or define it on first use.  
+
+space is limited. For example, in a table or chart. If the full word can’t fit, link to a note under the table or chart with the full word. 
+
+Abbreviated content is then accessible to all users. 
+
+Common Department of Agriculture, Fisheries and Forestry (DAFF) abbreviations include: 
+
+- app (for ‘application’) 
+- com (for ‘commodity’) 
+- vet (for ‘veterinary’) 
+- eCert (for ‘ecertification’) 
+- ePhyto (for ‘electronic phytosanitary certificate’) 
+- cont. (for ‘continued’) 
+- Dec (for ‘December’) 
+- e.g. (for ‘example’). 
+
+Include the full word for an abbreviation before or after you first use it on a page. Or, add an abbreviation and the full word to a glossary and link to it. 
+
+For more information, see: 
+
+- Abbreviations 
+- DAFF Style guide (internal DAFF staff only). 
+
+## Acronyms and initialisms 
+
+An acronym is a shortened form and a type of abbreviation. It’s made up of the first letter of each word in a name or noun. An acronym replaces the full name and is pronounced as a word.  
+
+An initialism is similar to an acronym. It’s also made up of the first letter of each word in a name. An initialism replaces the full name but is pronounced as separate letters. You can even have a combination of an acronym and initialism. 
+ 
+Use an acronym or initialism only if users are familiar with it and will understand it. It can make content shorter and faster to read. 
+
+Use capital letters for an acronym or initialism generally. Don’t use a full stop after an acronym or initialism. Don’t use an acronym or initialism in a heading. You don’t need to spell out  
```

Comment:
What is this "You don’t need to spell out " at the end?

Reply:
The error is in the original copy. I'm happy to re-write, so that it makes sense.